workers/unity/Packages/io.improbable.gdk.core/Worker/ConnectionHandlers/MockConnectionHandler.cs

```diff
@@ -211,6 +290,29 @@ public RemoveEntityTemplateDynamicHandler(EntityTemplate template, long entityId
             }
         }
 
+        private class MockOutgoingCommandHandler : IOutgoingCommandHandler
+        {
+            private long nextRequestId = 1;
+
+            private readonly Action<long, Type, ICommandRequest> onSendCommand;
+
+            internal MockOutgoingCommandHandler(Action<long, Type, ICommandRequest> onSendCommand)
```

Comment:
Why does this need a closure rather than accepting the `MockConnectionHandler` concretely? There's now a level of indirection between them that isn't necessarily required.